src/components/home/right-panel.tsx

```
@@ -0,0 +1,60 @@
+              </AvatarFallback>
+            </Avatar>
+            <div className="flex flex-col">
+              <p>{conversationName}</p>
+              {selectedConversation.isGroup && (
+                <GroupMembersDialog selectedConversation={selectedConversation} />
+              )}
+            </div>
+          </div>
+
+          <div className="flex items-center gap-7 mr-5">
+            <a href="/video-call" target="_blank">
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)